vite.config.ts

```diff
@@ -71,6 +79,12 @@ export default defineConfig(({ mode }) => {
   }
 })
 
+function scrivitoTenantIsMissing(env: Record<string, string>): boolean {
```

Comment:
* [ ] FYI Splitting the check and the error between the top and bottom of the module is hard to read. We could use an assert function that throws by itself.